src/studio.c

```diff
@@ -1945,12 +1950,18 @@ static void studioTick()
             memcpy(tic->ram.font.data, impl.systemFont.data, sizeof(tic_font));
         }
 
-        data
-            ? tic_core_blit_ex(tic, scanline, overline, data)
-            : tic_core_blit(tic);
+        if(isRecordFrame())
```

Comment:
Should we be handling the edge case where the frame record format is equal to the blitting format? On modern computers I don't think it makes much of a difference, but...

Reply:
I think we don't have to handle the edge case, it looks more clear atm

Reply:
OK